tests/test_tui_api.py

```diff
@@ -5,12 +5,9 @@
 from ansys.fluent.core.services.datamodel_tui import TUIMenu
 
 
-@pytest.mark.skip("randomly failing due to missing transcript capture")
+@pytest.mark.skip("Failing in github")
 def test_report_system_proc_stats_tui(new_solver_session, capsys) -> None:
-    new_solver_session.transcript.start()
-    # Issue: Transcript missing for the first TUI command
-    new_solver_session.solver.tui.report.system.sys_stats()
-    new_solver_session.solver.tui.report.system.sys_stats()
+    new_solver_session.tui.report.system.sys_stats()
```

Comment:
Would using the settings API equivalent make any difference? Both TUI and settings API work on my end (v232 and above)
```suggestion
    new_solver_session.results.report.system.sys_statistics()
```